doc/api/events.md

```diff
@@ -307,6 +307,45 @@ to each.
 
 Returns `true` if the event had listeners, `false` otherwise.
 
+
+```js
+const EventEmitter = require('events');
+const myEmitter = new EventEmitter();
+
+// First listener
+myEmitter.on('event', function firstListener() {
+  console.log('Helloooo! first listener');
+});
+// Second listener
+myEmitter.on('event', function secondListener(arg1, arg2) {
+  console.log(`event with parameters ${arg1}, ${arg2}, in second listener`);
+});
+// Third listener
+myEmitter.on('event', function thirdListener(...args) {
+  let parameters = '';
+  args.forEach((arg) => {
+    parameters += `${arg}, `;
+  }
+  );
+  console.log(`event with parameters ${parameters}in third listener`);
+});
+
+console.log(myEmitter.listeners('event'));
+
+myEmitter.emit('event', 1, 2, 3, 4, 5);
+
+// Prints:
+// [
+//  [Function: firstListener],
+//  [Function: secondListener],
+//  [Function: thirdListener]
```

Comment:
Super tiny nit: the indentation seems off. To keep it aligned with the starting bracket, it needs another whitespace.